tools/generators/migrate-converged-pkg/index.ts

```diff
@@ -388,6 +393,43 @@ const templates = {
     .git*
     .prettierignore
   ` + os.EOL,
+  swcConfig: () => {
+    return {
+      $schema: 'https://json.schemastore.org/swcrc',
+      env: { targets: { chrome: '84', edge: '84', firefox: '75', opera: '73', safari: '14.1' }, bugfixes: true },
```

Comment:
Should be a supported version of Node under `targets` (https://swc.rs/docs/configuration/supported-browsers)?

Reply:
I don't believe we have any definitive node target established 🤔